portal/src/graphql/portal/CreateProjectScreen.tsx

```diff
@@ -185,6 +185,7 @@ function CreateProjectScreenContent(props: CreateProjectScreenContentProps) {
               <FormPhoneTextField
                 parentJSONPointer=""
                 fieldName="phone_number"
+                initialCountry="auto"
```

Comment:
I suggest we add `geoIPCountryCode: String` to Viewer, and use that as the initialCountry to this phone field. That saves us from adding a new endpoint.